client/state/data-layer/wpcom/me/blocks/sites/test/index.js

```diff
@@ -0,0 +1,53 @@
+/** @format */
```

Comment:
I used to write a lot of those tests before, but have stopped recently. The reason is that such tests give us very little value - they only test whether a function that return X actually returns X, which feels obsolete and unnecessary. We know that `1 = 1`, and we don't need a test to verify that statement. So I'd suggest completely removing those tests.

Reply:
I think the tests would have little value if they were re-using the action creators, but since the data layer is creating its own actions, these do have value, as they validate the shape of each dispatched action. Typos are easy to make.

Reply:
I have no strong preferences, so I'm happy to go with yours.